reference/promise-types/custom.markdown

```diff
@@ -371,6 +371,13 @@ You can also include log messages in the JSON data:
 
 (Formatted here for readability, JSON data must still be on a single line in protocol).
 
+The JSON based protocol also supports the use of custom bodies.
+Custom bodies are sendt as a JSON objects within the attribute. E.g.
+
+```
+"attributes": { "person": { "name": "alice", "age": 27 } }
+```
```

Comment:
Maybe you should add an example policy here, to make it really clear. with 1 body, 1 bundle, 1 promise, it can be your groups promise :)